src/kvstore/RocksEngine.cpp

```diff
@@ -330,14 +330,20 @@ void RocksEngine::removePart(PartitionID partId) {
 
 std::vector<PartitionID> RocksEngine::allParts() {
     std::unique_ptr<KVIterator> iter;
-    static const std::string prefixStr = NebulaKeyUtils::systemPartsPrefix();
+    static const std::string prefixStr = NebulaKeyUtils::systemPrefix();
```

Comment:
Maybe we can scan with prefix = kSystem + kSystemPart.

Reply:
But I put partition ID with kSystem